builds/MEASLES.md

```diff
@@ -6,10 +6,25 @@
 * This downloads the file `sequence.seq`
 * Open this file and remove the `.1`, `.2`, etc... from the accession numbers
 
+```
+cat sequence.seq | sed 's/.1$//g' | sed 's/.2$//g' > one.txt
+mv one.txt sequence.seq
+```
```

Comment:
1. Update list item text.
2. Indent the code block to be part of the list item.
3. Simplify command:
    1. Use `sed -i ''` for editing inline (no need for `one.txt`).
    2. Use `sed -e ` for multiple editing commands (no need for `cat` + 2 `sed` calls).

~~~suggestion
* Remove the `.1`, `.2`, etc... from the accession numbers in `sequence.seq`:

    ```
    sed -i '' -e 's/.1$//g' -e 's/.2$//g' sequence.seq
    ```
~~~